egs/swbd/s5c/local/swbd1_prepare_dict.sh

```diff
@@ -91,6 +89,6 @@ cat $dir/acronyms_raw.map | sort -u > $dir/acronyms.map
 pushd $dir >&/dev/null
 ln -sf lexicon5.txt lexicon.txt # This is the final lexicon.
 popd >&/dev/null
-rm $dir/lexiconp.txt
+rm -f $dir/lexiconp.txt
```

Comment:
I prefer
`rm $dir/lexiconp.txt 2>/dev/null`
to suppress the warning.
Sometimes people make files non-writable because they don't want them accidentally deleted.